pkg/node/manager/manager.go

```diff
@@ -532,3 +539,42 @@ func (m *Manager) DeleteAllNodes() {
 	m.nodes = map[nodeTypes.Identity]*nodeEntry{}
 	m.mutex.Unlock()
 }
+
+// StartNeighborRefresh spawns a controller which refreshes neighbor table
+// by sending arping periodically. Linux keeps an arp entry in reachable
+// state for (0.5 ~ 1.5) * base_reachable_time_ms, default by 15 to 45 seconds:
+// https://elixir.bootlin.com/linux/v5.7.19/source/net/core/neighbour.c#L113
+// We do the refresh in a similar way.
+func (m *Manager) StartNeighborRefresh(nh datapath.NodeHandler) {
+	baseNeighborRefreshInterval := option.Config.NeighborRefreshBaseInterval
+	if baseNeighborRefreshInterval <= 0 {
+		baseNeighborRefreshIntervalStr, err := sysctl.Read("net.ipv4.neigh.default.base_reachable_time_ms")
+		if err != nil {
+			baseNeighborRefreshInterval = 30000 * time.Millisecond
+		} else {
+			baseNeighborRefreshIntervalU32, err := strconv.ParseUint(baseNeighborRefreshIntervalStr, 10, 32)
+			if err != nil {
+				baseNeighborRefreshInterval = 30000 * time.Millisecond
+			} else {
+				baseNeighborRefreshInterval = time.Duration(baseNeighborRefreshIntervalU32) * time.Millisecond
+			}
+		}
+	}
```

Comment:
Style nit: we can shorten the `baseNeighborRefreshInterval` variable name to `interval` or similar. IMO, we don't need a overly verbose variable name in such a small scope. I feel shortening will make the code slightly easier to read.